kube/app.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: wattx-gke-snapshooter
+spec:
+  replicas: 1
```

Comment:
minor but you could add:
```
revisionHistoryLimit: 1
```
to limit number of storing revisions to 2 (current & old - in case of rollback)

(otherwise you have to remove them manually)